lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -1629,3 +1645,641 @@ void SpecialFunctionHandler::handleAddBPFCall(ExecutionState &state,
   /* Terrible HACK. This has to go */
   state.bpf_calls++;                                      
 }
+
+void SpecialFunctionHandler::handleMemcmp(ExecutionState &state,
```

Comment:
For each such function, I would add in the comments the exact expression you are creating. You have it for some, do it for all